aspnetcore/security/anti-request-forgery.md

```diff
@@ -221,14 +218,14 @@ In traditional HTML-based applications, antiforgery tokens are passed to the ser
 
 ### AngularJS
 
-AngularJS uses a convention to address CSRF. If the server sends a cookie with the name ``XSRF-TOKEN``, the Angular ``$http`` service will add the value from this cookie to a header when it sends a request to this server. This process is automatic; you don't need to set the header explicitly. The header name is ``X-XSRF-TOKEN``. The server should detect this header and validate its contents.
+AngularJS uses a convention to address CSRF. If the server sends a cookie with the name `XSRF-TOKEN`, the Angular `$http` service will add the value from this cookie to a header when it sends a request to this server. This process is automatic; you don't need to set the header explicitly. The header name is `X-XSRF-TOKEN`. The server should detect this header and validate its contents.
```

Comment:
Please create an issue to update this section. Most customers have moved from AngularJS to Angular.

Reply:
Have both rather than update.